UPGRADING

```diff
@@ -240,6 +240,7 @@ PHP 8.3 UPGRADE NOTES
 
 - DOM:
   . Added DOMNode::contains() and DOMNameSpaceNode::contains().
+  . Added DOMElement::className.
```

Comment:
I put this in the "new functions" section, because I didn't know where to put it: there is no "new properties/fields" section...